backend/app/api/game.py

```diff
@@ -157,10 +157,25 @@ def get_tile():
 
     return tile
 
-
+"""function to filter chat messages"""
 def filter_message(data: Dict[str, str]) -> Dict[str, str]:
-    """function to filter chat messages"""
 
+    bannedWords = ["shit","fuck","crap","bitch"]
+    censoredSentence = []
+    " ".join(censoredSentence)
+    
+    data = {
```

Comment:
defining `data` here will overwrite the data variable being passed into the function.
The `data` dictionary already contains the data that you need to edit. You should be able to pull out the input sentence from `data['message']`.